modules/android/vsandroid_vcxproj.lua

```diff
@@ -78,6 +97,11 @@
 
 	premake.override(vc2010.elements, "configurationProperties", function(oldfn, cfg)
 		local elements = oldfn(cfg)
+
+		table.findAndRemove(elements, vc2010.characterSet)
```

Comment:
wrap those removes in a check for cfg.system == p.ANDROID or else a premake call can't deal with vsandroid and vs2010 both present

Reply:
Sure thing, [fixed](https://github.com/ifarbod/premake-core/commit/dfdaf2ca160b3a4adddc5743dccb16e0f0ddcc48).